### DISCUSSION TITLE
Refreshing IP Doesn't eliminate Newegg IPban

### DISCUSSION THREAD

**Discussion by l-u-n-c-h-b-o-x** [jef/streetmerchant#1418] [General]
tangentially related to streetmerchant, but with cookies and cache cleared, ipconfig/release then ipconfig/renew still results in the bot ban splashscreen on newegg.com. Any fixes?

**Comment by gigi2006** [comment]
it is not about your local IP but in the what you get from the provider. I do not know how it is with you, but here in Germany that is usually so that you get after a router reboot a new IP or stop in the evening the forced separation.

but that alone is probably not enough. work with user agents or random user agents.

ideal would be also a VPN connection which you can change again and again.

others prefer proxies, but I'm out, because these are also shared when free and when private then they are too expensive for me :)

**Reply by l-u-n-c-h-b-o-x** [reply]
They unbanned me automatically but that makes sense, thank you.

**Comment by CorgFather** [comment]
To add to what @gigi2006 said, if you're in the US, most home connections rarely assign new IPs, though they are generally dynamic. If you want to see your external IP, google "What is my IP." 

You're in luck, as VPN sales are common this time of year. Set up a VPN, repeat the search.
